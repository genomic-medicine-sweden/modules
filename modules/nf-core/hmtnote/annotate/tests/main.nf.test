nextflow_process {

    name "Test Process HMTNOTE_ANNOTATE"
    script "modules/nf-core/hmtnote/annotate/main.nf"
    process "HMTNOTE_ANNOTATE"
    config "./nextflow.config"
    tag "modules"
    tag "modules_nfcore"
    tag "hmtnote"
    tag "hmtnote/annotate"

    test("hmtnote test") {

        when {
            process {
                """
                input[0] = [ [ id:'test' ], // meta map
                        file(params.test_data['sarscov2']['illumina']['test_vcf'], checkIfExists: true)
                        ]
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
