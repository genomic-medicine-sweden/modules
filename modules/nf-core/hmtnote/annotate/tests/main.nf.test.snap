{
    "hmtnote test": {
        "content": [
            {
                "0": [
                    [
                        {
                            "id": "test"
                        },
                        "test_annotate.vcf:md5,6627d87e41823eb026deb8e196855163"
                    ]
                ],
                "1": [
                    "versions.yml:md5,8ee58af72515ea6d362a7721c3ba1340"
                ],
                "vcf": [
                    [
                        {
                            "id": "test"
                        },
                        "test_annotate.vcf:md5,6627d87e41823eb026deb8e196855163"
                    ]
                ],
                "versions": [
                    "versions.yml:md5,8ee58af72515ea6d362a7721c3ba1340"
                ]
            }
        ],
        "timestamp": "2024-02-13T11:43:41.894165395"
    }
}